synapseclient/core/download/download_async.py

```diff
@@ -220,104 +205,59 @@ def _pre_signed_url_expiration_time(url: str) -> datetime:
     Returns:
         A datetime in UTC of when the url will expire
     """
-    parsed_query: dict = parse_qs(urlparse(url).query)
-    time_made: str = parsed_query["X-Amz-Date"][0]
-    time_made_datetime: datetime.datetime = datetime.datetime.strptime(
-        time_made, ISO_AWS_STR_FORMAT
-    )
-    expires: str = parsed_query["X-Amz-Expires"][0]
-    return time_made_datetime + datetime.timedelta(seconds=int(expires))
+    parsed_query = parse_qs(urlparse(url).query)
+    time_made = parsed_query["X-Amz-Date"][0]
+    time_made_datetime = datetime.datetime.strptime(time_made, ISO_AWS_STR_FORMAT)
+    expires = parsed_query["X-Amz-Expires"][0]
+    return_data = time_made_datetime + datetime.timedelta(seconds=int(expires))
+    if return_data.tzinfo is None:
```

Comment:
When would this be None?

Reply:
I updated this to be:
```
    return_data = (
        time_made_datetime + datetime.timedelta(seconds=int(expires))
    ).replace(tzinfo=datetime.timezone.utc)
```

By default all datetime objects in Python are not TZ aware (Bad decision IMO), but this lets everything down stream know for certain what TZ this is expected to be.